dsa-solutions/lc-solutions/0500-0599/0555-split-concatenated-strings.md

```diff
@@ -0,0 +1,190 @@
+---
+id: split-concatenated-strings
+title: Split Concatenated Strings
+sidebar_label: 0555-split-concatenated-strings
+tags:
```

Comment:
Not good way 

```md
tags:
  - Greedy
  -	Array
  -	String
```

So, write in right format

Reply:
done